client/app/components/debugger/debugger.component.ts

```diff
@@ -50,6 +51,9 @@ export class DebuggerComponent {
             .map(fileContents => {
                 this.lines = fileContents.split('\n')
                     .filter((l, i) => this.sourceFunction && i >= (this.sourceFunction.lineStart - 1) && i < (this.sourceFunction.lineStart - 1 + this.sourceFunction.lineCount));
+                this.lines = this.lines.map(function(l) {
```

Comment:
Does the syntax highlighting still work when we have expressions/statements that span multiple lines?